src/org/omegat/core/search/Searcher.java

```diff
@@ -662,6 +681,55 @@ public boolean searchString(String origText, boolean collapseResults) {
         return true;
     }
 
+    // Interpret case changes in regex. Must be called _AFTER_ variables replacement!
+    private static String replaceCase(String txt) {
```

Comment:
Since this doesn't have anything specifically to do with search, I'd like to see it moved to a util class like `StringUtil` and have some unit tests added.

Reply:
Done.
For the moment I committed in github only small things, in order to check that I had correct write access. More will follow during next days.